src/modules/dashboard/components/TaskRequestWork/TaskRequestWork.tsx

```diff
@@ -30,21 +29,16 @@ const displayName = 'dashboard.TaskRequestWork';
 // Can't seal this object because of withConsumerFactory
 interface Props {
   openDialog: OpenDialog;
-  task: TaskType;
-  history: any;
+  task: AnyTask;
+  history: History;
 }
 
-const TaskRequestWork = ({
-  task: { colonyAddress, draftId },
-  task,
-  history,
-}: Props) => {
+const TaskRequestWork = ({ task: { id: draftId }, task, history }: Props) => {
   const { username, walletAddress } = useLoggedInUser();
 
-  const transform = useCallback(mergePayload({ colonyAddress, draftId }), [
-    colonyAddress,
-    draftId,
-  ]);
+  const [sendWorkRequest] = useCreateWorkRequestMutation({
+    variables: { input: { id: draftId } },
```

Comment:
We're still a bit inconsistent with taskIds/draftIds. No idea whether we should resolve that now. Probably not